src/debug.c

```diff
@@ -459,6 +460,10 @@ void debugCommand(client *c) {
 "    Setting it to 0 disables expiring keys in background when they are not",
 "    accessed (otherwise the Redis behavior). Setting it to 1 reenables back the",
 "    default.",
+"QUICKLIST_PACKED_THRESHOLD <size>",
```

Comment:
SET-ACTIVE-EXPIRE
SET-SKIP-CHECKSUM-VALIDATION
QUICKLIST_PACKED_THRESHOLD
should we use `-` here?